src/pages/workspaces/workspace/Data.js

```diff
@@ -216,7 +216,7 @@ const BucketContent = _.flow(
     withErrorReporting('Error loading bucket data'),
     Utils.withBusyState(setLoading)
   )(async (targetPrefix = prefix) => {
-    const { items, prefixes } = await Ajax(signal).Buckets.list(googleProject, bucketName, targetPrefix)
```

Comment:
This is the key change. Previously, `list` was capped at 1k and did not support pagination, so this switches to use `listAll` which deals with pagination.

Reply:
How is the pagination exposed in the UI?

Reply:
It's handled internally by gathering all of the results for the available pages. It's not displayed to the user.

Reply:
For additional context, I think exposing the pagination is the right thing to do long-term. But this fix is motivated by a fairly high-priority blocker for GP so that's why I chose the quicker approach here

Reply:
I'm a bit concerned about what happens when there's a very large number of objects in the bucket. Like if there's 100k workflow folders or something like that. I feel like it may not be long until we get a ticket for the files tab being unresponsive. We also reload the objects list every time a file is uploaded or deleted, so those may become less usable with a large number of files.

But that seems a less severe issue than hiding objects, so I'm on board with starting with the quick fix.